compiler/test/input/memoryBase/asserts.gr

```diff
@@ -1,4 +1,4 @@
-/* grainc-flags --memory-base 0x110000 */
+@compileFlags("--memory-base", "0x110000")
```

Comment:
I agree, I generally prefer `=` for long option syntax. Are you suggesting to just replace the instances of `@compileFlags` where this is done or changing the CLI to completely prohibit long options without `=`? Because `--arg value` is technically legal as well according to `getopt_long`/the GNU long option convention

Reply:
I'm suggesting that command line flags are not the same as configuration passed via `@compile` attribute. CLI flags should be left alone and the attribute stuff should be adjusted to my suggestions and then they should be normalized.